figo/models.py

```diff
@@ -17,6 +17,8 @@ def from_dict(cls, session, data_dict):
     def __init__(self, session, **kwargs):
         self.session = session
         for key, value in kwargs.items():
+            if isinstance(value, unicode):
```

Comment:
Arg. I don't like this solution. Can't we just fix all `__str__` methods?

Please have a look at [this](https://github.com/figo-connect/python-figo/blob/master/figo/models.py#L663) too :fearful:  :scream:

Reply:
You're probably right.

The link is hideous, and we decided to not support 2.6 anymore.